server/src/test/java/org/opensearch/index/shard/IndexShardTests.java

```diff
@@ -3500,6 +3501,27 @@ public void testReadSnapshotConcurrently() throws IOException, InterruptedExcept
         closeShards(newShard);
     }
 
+    /**
+     *  Test that latestReplicationCheckpoint returns null only for docrep enabled indices
+     */
+    public void testReplicationCheckpointNullForDocRep() throws IOException {
```

Comment:
nit - pls move to SegmentReplicationIndexShardTests.

Reply:
Done